docs/apis-tools/spring-zeebe-sdk/configuration.md

```diff
@@ -310,28 +437,54 @@ Read more about this feature in the [job streaming documentation](/apis-tools/ja
 
 To enable job streaming on the Zeebe client, you can configure it:
 
-```properties
-zeebe.client.default-job-worker-stream-enabled=true
+```yaml
+camunda:
+  client:
+    zeebe:
+      defaults:
+        stream-enabled: true
+```
+
+This also works for every worker individual:
```

Comment:
Do you mean this also works for every job worker individually?

Reply:
Yes, this is my understanding.

Reply:
:upside_down_face: Should we fix the typo then?

```suggestion
This also works for every worker individually:
```